applications/nrf5340_audio/src/audio/audio_sync_timer.c

```diff
@@ -152,4 +242,21 @@ uint32_t audio_sync_timer_curr_time_get(void)
 	return nrfx_timer_capture(&timer_instance, AUDIO_SYNC_TIMER_CURR_TIME_CAPTURE_CHANNEL);
 }
 
+void audio_sync_timer_sync_led_on(void)
+{
+	nrfx_timer_compare(&timer_instance, AUDIO_SYNC_TIMER_SYNC_LED_OFF_CMPR_CHANNEL, 0, true);
```

Comment:
Why is it necessary to set the OFF channel compare value to 0 here?

Reply:
When this the "pause" button is pressed there OFF channel has an compare value and if this value is not set to 0 the LED will turn off after the GPIO is set by the trigger. As the OFF compare value is set forward in time.